docs/source/configuration.rst

```diff
@@ -478,7 +478,7 @@ In *.sqlfluffignore*:
 .. code-block::
 
     target/
-    dbt_modules/
+    dbt_packages/
```

Comment:
```suggestion
    # dbt <1.0.0
    dbt_modules/
    # dbt >=1.0.0
    dbt_packages/
```
Let's keep both here as a lot of people haven't moved to dbt 1.0.0 yet

Reply:
There's also a `.sqlfluffignore` file at `plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/.sqlfluffignore`. Could you update it to:
```
target/
# dbt <1.0.0
dbt_modules/
# dbt >=1.0.0
dbt_packages/
macros/
```
to be consistent with the docs?